src/vs/workbench/browser/quickaccess.ts

```diff
@@ -45,3 +47,54 @@ export function getQuickNavigateHandler(id: string, next?: boolean): ICommandHan
 		quickInputService.navigate(!!next, quickNavigate);
 	};
 }
+
+export function getSelectionTextFromEditor(allowUnselectedWord: boolean, activeEditor: IEditor): string | '' {
```

Comment:
I think this layer is fine for now, but since it's being used for search quick access AND search view, it probably should be somewhere else in the Editor realm of things.

Reply:
Can you add a comment that says:

```
TODO: Find a better place for this function since it's shared between quick access and the search view
```

Reply:
Sure! I was thinking of adding a comment too

Reply:
Done